gunicorn/tox.ini

```diff
@@ -3,7 +3,7 @@ isolated_build = true
 minversion = 2.0
 basepython = py38
 envlist =
-    py{27,38}-{19.2,19.9,latest}
+    py27-19.9,py38-{19.9,20.1}
```

Comment:
Because gunicorn 20 dropped python 2 support so the environment setup failed here: https://github.com/DataDog/integrations-core/blob/1ea6327b38abf70cd21c3fa3825da4c17853fbab/gunicorn/tests/conftest.py#L77-L86

I didn't want to spend time understanding why installing that is necessary for the test setup, so I thought it made sense to simply drop it for now.

Reply:
Sounds good to me. Did not know gunicorn 20 dropped py2